test/objspace/test_ractor.rb

```diff
@@ -0,0 +1,16 @@
+require "test/unit"
+require "objspace"
+
+class TestObjSpaceRactor < Test::Unit::TestCase
+    def test_tracing_does_not_crash
+        ObjectSpace.trace_object_allocations do
+            r = Ractor.new do
```

Comment:
Now Ractor's test should be in a different process.

Reply:
ahah, with the `assert_ractor` helper, I see. Thanks!